src/game/models/card.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+
+class Card(models.Model):
+    # ForeignKeys
+    game = models.ForeignKey('game.Game', on_delete=models.CASCADE, related_name='cards')
+    base_id = models.ForeignKey('game.BaseCard', on_delete=models.CASCADE)
```

Comment:
As jou field se naam `base_id` is, dan gaan jou model 'n field he met die naam `base_id_id`... I think you get my point... -_-